IronFrame.Test/LocalTcpPortManagerTests.cs

```diff
@@ -130,5 +130,28 @@ public void ThrowWardenExecptionIfFirewallManagerFails()
                 Assert.IsType<Exception>(exception);
             }
         }
+
+        public class CreateFirewallRule : LocalTcpPortManagerTests
+        {
+            [Fact]
+            public void DelegatesToFirewallManager()
+            {
+                var tcpPortManager = new LocalTcpPortManager(FirewallManager, NetShRunner);
+                var firewallRuleSpec = Substitute.For<FirewallRuleSpec>();
```

Comment:
I guess it could be. We didn't have a strong opinion to do it one way or the other, but we thought it'd be nice to be consistent and substitute or dependencies that aren't being tested.